libs/ui/src/examples/Fetch/E1_FetchAxios.tsx

```diff
@@ -11,8 +12,19 @@ export const FetcAxios = () => {
   const [blogs, setBlogs] = useState([])
```

Comment:
You can specify the type of state. Because we already know what it will be after fetching.

```suggestion
  const [blogs, setBlogs] = useState<Blog[]>([])
```